test/unit/org/apache/cassandra/sensors/SensorsReadTest.java

```diff
@@ -258,6 +277,39 @@ private void assertRequestAndRegistrySensorsEquality(Context context)
         assertThat(registrySensor).isEqualTo(localSensor);
     }
 
+    private void assertRegistrySensorAndResponseEquality()
+    {
+        Sensor localSensor = getThreadLocalRequestSensor();
+        assertThat(localSensor.getValue()).isGreaterThan(0);
+
+        assertRegistrySensorAndResponseEquality(localSensor.getValue());
+    }
+
+    private void assertRegistrySensorAndResponseEquality(double ...requestSensorValues)
+    {
+        String readBytesTotalHeader = Type.READ_BYTES.name() + "_TOTAL";
+        double readBytesSum = 0;
+        int messageIndex = 0;
+        for (Message message : capturedOutboundMessages)
+        {
+            assertThat(message.header.customParams()).isNotNull();
+            assertThat(message.header.customParams()).containsKey(Type.READ_BYTES.name());
```

Comment:
Why not reusing the `SensorsCustomParams` static fields as keys here?

Reply:
I think it is part of the test objective to verify that  that the param value is indeed `READ_BYTES` (now updated to `READ_BYTES_[REQUEST|TABLE]` just in case if `SensorsCustomParams.READ_BYTES` is accidentally updated.

Reply:
Fair enough!